Tetiana_code/HW7/calculate_area_triangle_circle_rectangle1.py

```diff
@@ -0,0 +1,37 @@
+import math
```

Comment:
good job
look at this implementation
import math

def rectangle_area(length, width):
    return length * width

def triangle_area(base, height):
    return 0.5 * base * height

def circle_area(radius):
    return math.pi * radius ** 2

print("Make your choice: "1. Rectangle,  2. Triangle, 3. Circle")

choice = input("Enter your choice (1/2/3): ")

if choice == "1":
    length = float(input("Enter the length of the rectangle: "))
    width = float(input("Enter the width of the rectangle: "))
    area = rectangle_area(length, width)
    print(f"The area of the rectangle is {area:.2f}")
elif choice == "2":
    base = float(input("Enter the base of the triangle: "))
    height = float(input("Enter the height of the triangle: "))
    area = triangle_area(base, height)
    print(f"The area of the triangle is {area:.2f}")
elif choice == "3":
    radius = float(input("Enter the radius of the circle: "))
    area = circle_area(radius)
    print(f"The area of the circle is {area:.2f}")
else:
    raise ValueError("Invalid choice entered. Please enter a valid choice (1/2/3)")